portfolio/src/main/java/com/google/sps/data/Video.java

```diff
@@ -19,6 +20,7 @@
     private boolean isPublic;
     // Fetch when analyzing.
     private String captions;
+    private VideoException error = null;
```

Comment:
I thought we decided on an approach where instead of an embedded exception, any errors will be thrown and caught by the user.